docs/actions/append_report.md

```diff
@@ -13,15 +15,16 @@ Priority helps with sorting phases and table content by becoming part of the fil
 
 **phase:** *required*, *no default:*  
 The phase, for which we generate the report. Currently, there are three phases:
+
 - install
 - test
 - report
 
 **job:** *not required*, *default:* -  
-The name of the job, e.g, "phpunit", "codeception" etc.
+The name of the job, e.g. "phpunit", "codeception" etc.
 
 **title:** *not required*, *default:* -  
-The title of the current matrix run. This is usually defined in the job specific testplan, e.g, "unit_tests", "integration_tests" or "shop_setup".
+The title of the current matrix run. This is usually defined in the job-specific testplan, e.g. "unit_tests", "integration_tests" or "shop_setup".
```

Comment:
Last time in the supermarket: "Sammeln Sie Punkte?" -- "Nee, Kommas"
`The title of the current matrix run. This is usually defined in the job-specific testplan, e.g., "unit_tests", "integration_tests", or "shop_setup".`